src/psij/web/js/IntroBlurb.js

```diff
@@ -0,0 +1,39 @@
+var VC = VC || {};
+
+VC.IntroBlurb = Vue.component("intro-blurb", {
```

Comment:
You may not have been at the meeting.  I was told that we would have different deployments and that people wanted to be able to change the text without having to make changes in the code, other than a config file.  The idea was that the config file could be changed to accomodate any text string changes OR you could have one config file for PSI/J and another one for SDK and another one for another use case.

Reply:
It's possible that things weren't that clear during the meeting. I apologize for that. We already have different deployments and we don't want to maintain two codebases, which we don't. But changing html or changing a .js file, that is still changing some form of code. So again, let's try to keep this simple with text in a single place rather than spread around.

Reply:
What I meant was just keep all the text in one file.  It doesn't have to be customization.js.  It could be any other config file, whatever you want me to call it.  The reason they want it in 1 file is because that way it's easier to make text changes because it's all in one place.  Also, they want to be able to specify which machines to show.  And, as another requirement, they told me they want to be able to do more than 1 or 2 deployments.  And so, if we do it in the HTML file, we'll have to add a new IF statement for every place where we do a text replacement, whenever we add a new deployment.  However, if all the text is in one config file, we just add or create a new config file.